build/charts/theia/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+name: theia
+type: application
+displayName: Theia
+home: https://antrea.io/
+version: 0.1.0-dev
+appVersion: 0.1.0-dev
```

Comment:
I think we need to update these before we release?

Reply:
Yes. I think we will need to update these in the release branch.